vendor/DOtherSide/lib/src/DOtherSide.cpp

```diff
@@ -193,12 +179,12 @@ void dos_qtwebview_initialize()
 
 void dos_qguiapplication_try_enable_threaded_renderer()
 {
-    if(QSysInfo::kernelType() == "darwin" && QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
+    if(QSysInfo::kernelType() == QLatin1String("darwin") && QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
```

Comment:
wrapping raw listerals into `QLatin1String` makes sense only in case of `QT_NO_CAST_FROM_ASCII` defined. Otherwise there is no benefit of that, so I would keep it simple as it was. (https://doc.qt.io/qt-5/qlatin1string.html#details)

Reply:
Hmm, it makes no sense to me bc QString has also overloaded `operator==` for `const char*`, no conversion is needed. Is clazy complaining about that?

Reply:
Yeah it does complain... I wonder why then 🤔

Reply:
If it complains, just wrap it up as you proposed to suppress it, I'm fine with that :)

Reply:
> bool QString::operator==(const char *other) const
> The other const char pointer is converted to a QString using the fromUtf8() function.

Reply:
Mystery solved :)